Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:22-alpine3.21
```

Comment:
I'm not sure if this node version is supported yet or if I need to fall back to an older version.